<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAnimationDialog</class>
 <widget class="QDockWidget" name="CAnimationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>102</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>609</width>
    <height>102</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>524287</height>
   </size>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::AllDockWidgetAreas</set>
  </property>
  <property name="windowTitle">
   <string>Animation control</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0" colspan="11">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="currentSpinBox">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="startSpinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="loopCheckBox">
        <property name="text">
         <string>Loop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="inPlaceCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>In place</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="incPosCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Increment pos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="endSpinBox">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="playlistToolButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Use the PlayList</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="object_viewer.qrc">
        <normaloff>:/icons/ic_nel_animset.png</normaloff>:/icons/ic_nel_animset.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QToolButton" name="mixerToolButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Use the Mixer</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="object_viewer.qrc">
        <normaloff>:/images/mixer.png</normaloff>:/images/mixer.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QToolButton" name="startPushButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="object_viewer.qrc">
        <normaloff>:/images/skip-backward.png</normaloff>:/images/skip-backward.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QToolButton" name="backwardPushButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="object_viewer.qrc">
        <normaloff>:/images/seek-backward.png</normaloff>:/images/seek-backward.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QToolButton" name="playPushButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Play/Pause</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="object_viewer.qrc">
        <normaloff>:/images/play.png</normaloff>
        <normalon>:/images/pause.png</normalon>:/images/play.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QToolButton" name="stopPushButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Stop</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="object_viewer.qrc">
        <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QToolButton" name="forwardPushButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="object_viewer.qrc">
        <normaloff>:/images/seek-forward.png</normaloff>:/images/seek-forward.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QToolButton" name="endPushButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="object_viewer.qrc">
        <normaloff>:/images/skip-forward.png</normaloff>:/images/skip-forward.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="object_viewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>currentSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>currentSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
